### DISCUSSION TITLE
Datenübernahme 3.1: Datenpunkt Schlüssel Problem

### DISCUSSION THREAD

**Discussion by Bernd9000** [Zefau/ioBroker.jarvis#2005] [General]
@Zefau 
Bei der Umstellung von Version 3.0 auf 3.1 werden bei mir z.B.  diverse Werte im Popup nicht angezeigt und der Levelbody fehlt.
Ich hab dann im Expert Modus herausgefunden das es an der Variable "key" liegt.
Wenn ich diesen entferne wird alles wieder korrekt angezeigt.
Ich denke die Variable wird durch ein Geräteimport angelegt ?
Wird diese Variable in Version 3.1 noch benutzt ?

Beispiel:

 "temperatur": {
        "showState": true,
        "state": "0_userdata.0.Shelly.Eckgrundstück-Licht2.Temperatur",
        "action": "",
        "label": "",
        "bodyElement": "LastChangeBody",
        "stateKey": "temperatur",
        `"key": "temperature",`
        "unit": "°C"
      },

**Comment by Bernd9000** [comment]
Wenn die Variable `key` nicht umbedingt benötigt wird, würd ich Sie hier komplett in allen Datenpunkt-Schlüssel löschen.

**Comment by Bernd9000** [comment]
Hab das mal getestet, alle Datenpunkt-Schlüssel mit der Variable `key` entfernt und erneut von `3.0.15` nach `3.1.beta.7` importiert, läuft alles.
Nut in beta.8 funktioniert es jetzt nicht mehr

**Comment by Zefau** [comment]
mit v3.1.2 sollte alles funktionieren
